### PR TITLE
Typos Fixed: Articulation Improved for Project Outlook

### PR DESCRIPTION
**Pull request by engagepy**
- corrected typos
- corrected fragmentation error :)
- simplified actionable step messages where required
- document now reads without any articulation flaw, as it should be for such a great project

## What does this PR do?

Improves readability, fixes typos and holds the README.md at a higher standard of articulation that it was.

## Related PRs and Issues

Null

Closes: #[issue number that will be closed through this PR]

## Type of change

- [x] Improved README.md update only

## Please let us know test plan followed

Locally

Please describe the tests(if any). Provide instructions how its affecting the coverage.

## Checklist:
- [x] Have you read the [Contributing Guidelines on issues](https://keploy.io/docs/keploy-explained/contribution-guide/)?
- [x] My code follows the style guidelines of this project.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Comment by engagepy** [comment]
I have read the CLA Document and I hereby sign the CLA

File: README.md [hunk 1]
```diff
@@ -45,13 +45,13 @@ Keploy not only records API calls, but also records database calls and replays t
 > 🐰 **Fun fact:** Keploy uses itself for testing! Check out our swanky coverage badge: [![Coverage Status](https://coveralls.io/repos/github/keploy/keploy/badge.svg?branch=main&kill_cache=1)](https://coveralls.io/github/keploy/keploy?branch=main&kill_cache=1) &nbsp;
 
 ## 🚨 Here for  [Unit Test Generator](README-UnitGen.md) (ut-gen)? 
-Keploy's new launched world's first unit test generator(ut-gen) implementation of [Meta LLM research paper](https://arxiv.org/pdf/2402.09171), it understands code semantics and generates meaningful unit tests, aiming to:
+Keploy's newly launched world's first unit test generator(ut-gen) implementation of [Meta LLM research paper](https://arxiv.org/pdf/2402.09171), it understands code semantics and generates meaningful unit tests, aiming to:
```
Comment (shivamsouravjha) [inline README.md:48]: shouldn't it be Keploy has newly?

**Review by shivamsouravjha — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (engagepy) [reply README.md:48]: Sure.

File: README.md [hunk 2]
```diff
@@ -45,13 +45,13 @@ Keploy not only records API calls, but also records database calls and replays t
...
 
-- **Automate unit test generation (UTG)**: Quickly generate comprehensive unit tests and reduce the redundant manual effort.
+- **Automate unit test generation (UTG)**: Quickly generate comprehensive unit tests and reduce redundant manual effort.
 
-- **Improve edge cases**: Extend and improve the scope of tests to cover more complex scenarios that are often missed manually.
+- **Improve edge cases**: Extend and improve the scope of atuomated tests to cover more complex scenarios, often missed manually.
```
[the remark this answers is not in the document]
Comment (engagepy) [reply README.md:52]: On it.

File: README.md [hunk 3]
```diff
@@ -45,13 +45,13 @@ Keploy not only records API calls, but also records database calls and replays t
...
 
-- **Boost test coverage**: As codebase grows, ensuring exhaustive coverage should become feasible.
+- **Boost test coverage**: As codebases grow, ensuring exhaustive coverage should become feasible, this aligns with our mission.
```
[the remark this answers is not in the document]
Comment (engagepy) [reply README.md:55]: Noted.

File: README.md [hunk 4]
```diff
@@ -90,11 +90,11 @@ keploy test -c "CMD_TO_RUN_APP" --delay 10
 ## ✅ Test Coverage Integration
 To integrate with your unit-testing library and see combine test coverage, follow this [test-coverage guide](https://keploy.io/docs/server/sdk-installation/go/).
 
-> ####  **If You Had Fun:** Please leave a 🌟 star on this repo!  It's free, and you'll bring a smile. 😄 👏
+> ####  **If You Had Fun:** Please leave a 🌟 star on this repo!  It's free == a smile. 😄 👏
```
[the remark this answers is not in the document]
Comment (engagepy) [reply README.md:93]: Roger

File: README.md [hunk 5]
```diff
@@ -119,7 +119,7 @@ From Go's gopher 🐹 to Python's snake 🐍, we support:
 
 ## 🫰 Keploy Adopters 🧡
 
-So you and your organisation are using Keploy? That’s great. Please add yourselves to [**this list,**](https://github.com/orgs/keploy/discussions/1765) and we'll send you goodies! 💖
+If you or your organisation is using Keploy? That’s great. Please add yourselves to [**this list,**](https://github.com/orgs/keploy/discussions/1765) and we'll send you goodies! 💖
```
[the remark this answers is not in the document]
Comment (engagepy) [reply README.md:122]: Got it

**Comment by engagepy** [comment]
@shivamsouravjha suggestions incorporated. Thanks !

**Review by Sonichigo — APPROVED** [review]

**Review by khareyash05 — APPROVED** [review]

**Review by shivamsouravjha — APPROVED** [review]
